aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModule.java

```diff
@@ -1400,6 +1398,12 @@ public void conclude()
                 throw new ClusterException("failed to create cluster dir: " + clusterDir.getAbsolutePath());
             }
 
+            if (startupCanvassTimeoutNs <= leaderHeartbeatTimeoutNs)
```

Comment:
Did we want `startupCanvassTimeoutNs` to be a multiple of `leaderHeartbeatTimeoutNs`?

Reply:
Yes, you are right. I'll adjust the validation to ensure that canvass timeout is at least twice as large as the election timeout.

Reply:
Done.